assets/blocks/quiz/quiz-block/quiz-settings.js

```diff
@@ -239,6 +240,7 @@ const QuizSettings = ( {
 							onChange={ createChangeHandler( 'showQuestions' ) }
 						/>
 					</PanelRow>
+					<Slot name="SenseiQuizSettings" />
```

Comment:
This is very cool! I didn't know about this component. Looks like it also supports multiple "fills". So you can keep adding as much as you want! Nicely done @onubrooks !

One thing I wanted to clarify is the `SlotFillProvider`. I'm guessing you didn't add it anywhere because we already have it somewhere? Or in context of gutenberg it's already provided in WP editor?

Reply:
Yes @dadish Gutenberg already adds it in the editor!